br/pkg/task/stream.go

```diff
@@ -1057,7 +1057,11 @@ func restoreStream(
 		return errors.Annotate(err, "failed to fix index for some table")
 	}
 
-	// TODO split put and delete files
+	err = client.CleanUpKVFiles(ctx)
```

Comment:
Maybe use `defer` here so if we failed to restore (or the user manually cancels restore) we can clean the temp files as normal?

Reply:
if we want resume from some internal break. keep files can save some download time.
I think we better provider a manual cleanup files command.